### ISSUE TITLE
Host some examples saves to read somewhere

### ISSUE THREAD

**Issue by PureTryOut** [2009scape/api.2009scape.org#3]
I would like to get this API service running locally to see if I can make some changes, but for that I would need some player saves to work with. I'm not sure how to get those though, and it would be great if some example ones could be hosted somewhere for use with this repo.

**Comment by Ceikry** [comment]
You could trivially make some player saves by simply logging into random usernames on a local server. You would then have saves located in Server/data/players. 

It should be noted this API is planned for deprecation as a replacement is in the works already

**Comment by PureTryOut** [comment]
Oh that's good, I have some big complaints about the current one 😅

Is the source located somewhere? Preferably the Gitlab organisation 😉

**Comment by summerdevs** [comment]
> Oh that's good, I have some big complaints about the current one sweat_smile
> 
> Is the source located somewhere? Preferably the Gitlab organisation wink

https://gitlab.com/2009scape/2009scape

**Comment by PureTryOut** [comment]
That's just the gameserver no? I don't see this API replacement?

**Comment by summerdevs** [comment]
Were you not asking where you could find the source to create some player saves?

**Comment by PureTryOut** [comment]
Yes but not in the part you were quoting 🤔

**Comment by PureTryOut** [comment]
@Ceikry do you have a link to the upcoming replacement of this API server?

**Comment by Ceikry** [comment]
No, it's a project being undertaken by ryan and ryan alone, and he gets all Australian if I try to make a slightly better one than the one we have in the interim :joy:

**Comment by PureTryOut** [comment]
Oh, uh, interesting. I was thinking of making something myself since there didn't really seem to be happening anything.
And I'm going to anyway, even if it's not going to be used, I just want to do some programming and have some ideas. :stuck_out_tongue:
